multiapps-mta/src/main/java/org/cloudfoundry/multiapps/mta/resolvers/v2/ResourceLiveParameterResolver.java

```diff
@@ -0,0 +1,149 @@
+package org.cloudfoundry.multiapps.mta.resolvers.v2;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.cloudfoundry.multiapps.mta.model.DeploymentDescriptor;
+import org.cloudfoundry.multiapps.mta.model.Module;
+import org.cloudfoundry.multiapps.mta.model.ProvidedDependency;
+import org.cloudfoundry.multiapps.mta.model.Resource;
+import org.cloudfoundry.multiapps.mta.resolvers.Reference;
+import org.cloudfoundry.multiapps.mta.resolvers.ReferencePattern;
+
+public class ResourceLiveParameterResolver {
+
+    private static final String CONFIG = "config";
```

Comment:
name this constant something more descriptive, other than just the value it holds